flows/actions/testdata/add_contact_groups.json

```diff
@@ -47,17 +47,23 @@
                     "uuid": "33382939-babf-4982-9395-8793feb4e7c6",
                     "name": "Climbers",
                     "type": "group",
-                    "missing": true,
-                    "nodes": {
-                        "72a1f5df-49f9-45df-94c9-d86f7ea064e5": [
-                            "ad154980-7bf7-4ab8-8728-545fd6378912"
-                        ]
-                    }
+                    "missing": true
                 }
             ],
             "results": [],
             "waiting_exits": [],
-            "parent_refs": []
+            "parent_refs": [],
+            "problems": [
```

Comment:
I prefer `problem` to `warning`.. lots of warnings about things in the app already.. `problem` can have specific meaning and possibly db tables one day